build/postgres/Dockerfile

```diff
@@ -7,30 +7,32 @@ ARG DEBIAN_FRONTEND="noninteractive"
 ARG POSTGRES_VERSION
 
 # Update the Ubuntu and PostgreSQL repository indexes
-RUN apt-get update && \
-    apt-get -y -q install \
-        build-essential \
-        git-core \
-        libdb-dev \
-        libexpat1-dev \
+RUN apt-get update \
+    && apt-get install  --no-install-recommends -qy \
+        ca-certificates \
+        gcc \
+        git \
         libicu-dev \
-        libpq-dev \
-        libxml2-dev \
-        pkg-config \
-        postgresql-server-dev-${POSTGRES_VERSION}
+        make \
+        postgresql-client-${POSTGRES_VERSION} \
+        postgresql-server-dev-${POSTGRES_VERSION} \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN git clone https://github.com/metabrainz/postgresql-musicbrainz-unaccent.git /tmp/postgresql-musicbrainz-unaccent && \
-    git clone https://github.com/metabrainz/postgresql-musicbrainz-collate.git /tmp/postgresql-musicbrainz-collate && \
-    git clone https://github.com/omniti-labs/pg_amqp.git /tmp/pg_amqp
+# hadolint ignore=DL3003
```

Comment:
Would that be helpful to have a `.hadolint.yaml` in this repository?

Reply:
Depends on if you want to use it. I probably did this out of habbit. I can remove it if you want.

Reply:
Having such comment is alright but having a `.hadolint.yaml` would probably be even better.

Reply:
I don't think we want to apply this rule globally. It just does not make sense in this case. There are also lot more hadolint warnings mostly about quoting and switching directory. Maybe we should look at some sever or ignore them.